resources/definitions/teraform_inventor.def.json

```diff
@@ -0,0 +1,149 @@
+  {
+    "version": 2,
+    "name": "TeraForm Inventor",
+    "inherits": "fdmprinter",
+    "metadata":
+    {
+        "visible": true,
+        "author": "SpaceFactory",
+        "manufacturer": "SpaceFactory",
+        "file_formats": "text/x-gcode",
+        "platform": "teraform_inventor_platform.stl",
+        "platform_offset": [0, 0, -20],
+        "has_materials": true,
+        "has_machine_quality": true,
+        "machine_extruder_trains":
+        {
+            "0": "teraform_x_120"
+        }
+    },
+    "overrides":
+    {
+        "machine_name": { "default_value": "TeraForm Inventor" },
+        "machine_show_variants": { "default_value": false },
+        "material_diameter": { "default_value": 12 },
+        "material_bed_temp_wait": { "default_value": false },
+        "material_bed_temp_prepend": { "default_value": false },
+        "machine_width": { "default_value": 2400 },
+        "machine_depth": { "default_value": 1200 },
+        "machine_height": { "default_value": 1200 },
+        "machine_center_is_zero": { "default_value": true },
+        "machine_nozzle_tip_outer_diameter": { "default_value": 25 },
+        "machine_nozzle_head_distance": { "default_value": 15 },
+        "machine_firmware_retract": 
+        { 
+            "default_value": true,
+            "value": true
+        },
+        "machine_nozzle_size": { "default_value": 12 },
+        "machine_use_extruder_offset_to_offset_coords": { "default_value": false },
+        "layer_height":
+        {
+            "minimum_value": "1",
+            "minimum_value_warning":"2",
+            "maximum_value_warning":"10",
+            "settable_per_extruder": false
```

Comment:
No need to override this, it's already false in the fdmprinter.def.json

Reply:
Ok.

Reply:
Fixed